src/scales/scale.radialLinear.js

```diff
@@ -278,39 +285,43 @@ module.exports = function(Chart) {
 				fillText(ctx, scale.pointLabels[i] || '', pointLabelPosition, plFont.size);
 			}
 		}
+		ctx.restore();
 	}
 
 	function drawRadiusLine(scale, gridLineOpts, radius, index) {
 		var ctx = scale.ctx;
+		var valueCount = getValueCount(scale);
+		var pointPosition;
+
+		if (!gridLineOpts.circular && valueCount === 0) {
```

Comment:
Yes, we need to check `lineWidth` because it fallbacks to the default (`1.0`), but is the check of `color` needed?

Reply:
To be exact, shouldn't it skip only when `lineWidth === 0` and let fallback the default when it is other falsy values?

Reply:
I don't see where `lineWidth` fallback to `1.0`?

Reply:
For sure, I would skip drawing if `lineWidth` is `null` (user input) because we fallback only on `undefined`. But I think that `ctx.lineWidth = undefined;` (which could happen at line 302) is error prone because nobody knows exactly what's the current value of `ctx.lineWidth`, neither from where it comes from. It's also unstable because adding a plugin could change that value. All the same for `color` IMO, and everywhere we are explicitly setting `ctx` values.

Our defaults define valid values for both options, so if we receive a falsy value, that's mean the user manually set `undefined`, `null`, `0` or empty string, in which cases I would skip drawing.

But maybe I'm missing a valid use case which would break if we change that behavior?

Reply:
I'm fine with your suggested changes, but I'm carious if we should put the similar check for all canvas context-related property. For example, if a user set `Chart.defaults.global.defaultColor` to `undefined`, every element color will be unstable.